examples/maven-example/README.md

```diff
@@ -28,9 +28,9 @@ The following properties should be added to your environment or this example's [
 
 From the project directory, run the following:
 
-*Linux/Mac*: `./mvnw -s ./settings.xml spring-boot:run` or `./run.sh`
+*Linux/Mac*: `mvn -s ./settings.xml spring-boot:run` or `./run.sh`
```

Comment:
The `tmc-auth` examples still contain their wrappers:   
- [Maven Example](https://github.com/autonomic-tmc/tmc-auth/blob/kt/remove-maven-wrapper/examples/maven-example/mvnw)
- [Gradle Example](https://github.com/autonomic-tmc/tmc-auth/blob/kt/remove-maven-wrapper/examples/gradle-example/gradlew)

I spoke about removing the wrappers from the examples with @integerAndrew when the wrapper was removed as part of the transient license story. In the end, **we decided the wrapper should remain in the examples**.  This way if someone only wants to run the examples they do not need Maven or Gradle. (Maven would only be required if they needed to build tmc-auth.)

If you feel like we should remove the Maven wrapper completely from the examples, please discuss it with the team before doing it.  Let's make sure this is what we want to do.

If the vote is **yes** to remove:
-  Remove EVERYTHING related to the wrapper in the Maven example (wrapper files, wrapper directory, wrapper command, AND documentation)
- Remove the wrapper completely from the Gradle example as well

If the answer is **no**, then just revert this change using the below suggestion.
```suggestion
*Linux/Mac*: `./mvnw -s ./settings.xml spring-boot:run` or `./run.sh`
```